sumologic/util.go

```diff
@@ -294,3 +301,96 @@ func GetTimeRangeBoundary(tfRangeBoundary map[string]interface{}) interface{} {
 
 	return nil
 }
+
+func SuppressEquivalentTimeDiff(isRelative bool) func(k, oldValue, newValue string, d *schema.ResourceData) bool {
```

Comment:
A function header listing out examples of time differences we are suppressing would be helpful.

Reply:
yeah, a comment above the function.